js/app.js

```diff
@@ -1,59 +1,53 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
 var createReactClass = require('create-react-class');
-import { Board } from './board.js'
+var Cookies = require('universal-cookie');
 import { Header } from './header.js'
-import { Button } from './button.js'
-import { makeApiRequest } from './api_requests.js'
+import { Login } from './login.js'
+import { Dashboard } from './dashboard.js'
+import firebase from './firebase.js'
 
-
-var Body = createReactClass({
-    getInitialState: function() {
-        return {
-            wordList: []
-        };
-    },
-
-    createNewGame: function() {
-        var componentThis = this;
-
-        var url = "/api/game/create";
-        var method = "post";
-        var params = {};
-        var onSuccess = function(data) {
-            componentThis.setState({wordList: data.wordList});
-        };
-        var onError = function(error) {
-            console.log(error);
-        };
-
-        makeApiRequest(url, method, params, onSuccess, onError);
-    },
-
-    render: function() {
-        if (this.state.wordList.length === 0) {
-            return (
-                <div className="text-center">
-                    <Button buttonText="Create New Game" onClickFn={this.createNewGame} />
-                </div>
-            );
-        }
-
-        return (
-            <Board wordList={this.state.wordList}/>
-        );
-    }
-});
+const cookies = new Cookies();
 
 var App = createReactClass({
-    render: function() {
-        return (
-            <div>
-                <Header title="WordCodes"/>
-                <Body/>
-            </div>
-        );
-    }
+  getInitialState: function() {
+    return {
+      logged_in: false,
```

Comment:
so what if you were already logged in and open a new window? we should grab the token out of cookies and use that, right?

or is the code below already doing that?

Reply:
pretty sure this code already does that; when I was testing on local I had to delete cookies to be able to re-log back in (since there is no log out button currently)

it does, I think, cause the login screen to render briefly before moving onto the dashboard, so can probably be tidy-ed up a bit

Reply:
ah--thought I had responded to this before but couldn't find it. now I see its .. below already